### ISSUE TITLE
RapidSSL is untrusted in Firefox, but trusted according to SSLLabs

### ISSUE THREAD

**Issue by vith** [ssllabs/ssllabs-scan#727]
For example, https://nvidia.com produces `SEC_ERROR_UNKNOWN_ISSUER` in Firefox 67.0.4 (stable channel, Windows).

https://www.ssllabs.com/ssltest/analyze.html?d=nvidia.com thinks there are no trust issues

**Comment by sebastianas** [comment]
It says: "This server's certificate chain is incomplete. Grade capped to B."

**Comment by vith** [comment]
I think nvidia fixed their non-www site config now. It's no longer untrusted in Firefox for me. We'd need another old RapidSSL example to know if this is still a problem for SSLLabs.

Not sure how to find a new example so I'll just close this issue.

**Comment by sebastianas** [comment]
nvidia did not fix a thing. In firefox, please go to
preferences -> privacy & security -> view certificates
Click on Authorities, search for "Software Security Device". Once you remove it, you will see the warning on nvidia.com again. The site www.nvidia.com on the hand is properly configured and you go there firefox will add the intermediate certificate to its database and then you won't see the warning on nvidia.com (again).

**Comment by vith** [comment]
Ah, you are right. A fresh browser profile gave SEC_ERROR_UNKNOWN_ISSUER again.

**Comment by vith** [comment]
So I guess my issue was probably invalid from the start then. I thought the problem was originally that the CA was RapidSSL, a Symantec subsidiary, and had been removed from the Mozilla root store.

I don't recall the chain being incomplete when I opened the issue, but I guess I should have taken a screenshot of the results instead of just linking to them, since there's no way to see what the results were back in June anymore.

Sorry for the reopen/reclose noise.
